src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -232,3 +232,29 @@ The strings can also be accessed directly, outside of the context of IoC referen
 ```javascript
 that.msgLookup.lookup(value); // where value is either the string name or the key in the stringArrayIndex to lookup.
```

Comment:
After thinking it more, we probably should keep it since it does provide another alternative to retrieve message strings. But make it clear that the prerequisites for using this function is to have ```fluid.prefs.msgLookup``` as a grade component. This statement is useful when integrators create their own sub-components of ```prefsEditorLoader```.